### PR TITLE
[Do Not Review] Test PR

### PR DESCRIPTION
**Pull request by Avinash2Suresh**
- [ ] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

TODO: Include high-level description of the changes in this pull request.

### Why should this Pull Request be merged?

TODO: Justify why this contribution should be part of the project.

### What testing has been done?

TODO: Detail what testing has been done to ensure this submission meets requirements.

### CODE REVIEW

**Review by jayaseelan-james — APPROVED** [review]
Approving to test PR rules.

**Comment by Avinash2Suresh** [comment]
Recently, I noticed a PR where merging was enabled without the code-owners' approval. To verify, I raised a test PR, but it seems that's not the case here. Therefore, I am closing this PR.

**Comment by bkeryan** [comment]
@Avinash2Suresh @jayaseelan-james I think this was a side effect of the https://github.com/CasperWA/push-protected action, which edits the repo's branch protection rules to temporarily unprotect reviews. @dixonjoel noticed this and already restored the settings. I filed https://github.com/ni/measurement-plugin-python/issues/940 about preventing this in the future.

**Pull request closed without merging**
